packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -63,7 +69,7 @@ type TestDataType = {
   name: string;
   description: string;
   subRows?: TestDataType[];
-};
+} & Record<string, unknown>;
```

Comment:
Why you needed to add this?

Reply:
I guess the alternative is to add `booleanValue?: boolean`. I just figure it is easier to extend the tests by basing it on the lowest common data type the table may take.